.github/workflows/prepare_release.yml

```diff
@@ -37,16 +39,19 @@ jobs:
             -v ${{ inputs.version }}
             -d ${{ inputs.date }}
 
+      - name: Save static code analysis
+        run: bandit -r . -x ./tests,./scripts,./build -f txt -o static_code_analysis.txt --exit-zero
+
```

Comment:
for all of these can we manually trigger it and see what the code analysis looks like? I've had trouble before where some files weren't being filtered out properly so I just want to make sure